### DISCUSSION TITLE
Why is publishing workflow gone from insiders?

### DISCUSSION THREAD

**Discussion by hellt** [squidfunk/mkdocs-material#5279] [Q&A] [answered]
Hi,
I noticed that publish.yml has gone from insiders. Was it intentional?
If yes, what is the recommended procedure to build insiders container and pushing it to repo? Doing it yourself?

**Comment by squidfunk** [comment, marked as answer]
Yes, intentional, as this is now handled in `build.yml` as introduced in https://github.com/squidfunk/mkdocs-material/pull/5102 to allow for multi-platform builds (if desired).

**Reply by squidfunk** [reply]
The workflow is the same – set up `GHCR_TOKEN` etc. [as mentioned in the docs](https://squidfunk.github.io/mkdocs-material/insiders/getting-started/#with-docker) and create a release.

**Reply by hellt** [reply]
thanks @squidfunk
